frontend/src/scenes/authentication/signup/verify-email/verifyEmailLogic.ts

```diff
@@ -34,7 +34,10 @@ export const verifyEmailLogic = kea<verifyEmailLogicType>([
                         await api.create(`api/users/verify_email/`, { token, uuid })
                         actions.setView('success')
                         await breakpoint(2000)
-                        window.location.href = '/'
+
+                        const nextUrl = router.values.searchParams['next']
+
+                        window.location.href = nextUrl || '/'
```

Comment:
This could be a new account, so if they haven't onboarded this will redirect to products and lose the next - does that mean it'll be lost?